virttest/utils_v2v.py

```diff
@@ -718,6 +723,23 @@ def v2v_cmd(params):
 
     # Construct a final virt-v2v command
     cmd = '%s %s' % (V2V_EXEC, options)
+    logging.debug(cmd)
+
+    # To deal with inputs during conversion
+    if custom_inputs or (sasl_user and sasl_pwd):
+        v2v_log = os.path.join(data_dir.get_tmp_dir(), 'v2v_log')
+        v2v_session = aexpect.ShellSession(command=cmd,
+                                           output_func=utils_misc.log_line,
+                                           output_params=(v2v_log,))
+        remote.handle_v2v_prompts(v2v_session, sasl_user, sasl_pwd, timeout=v2v_timeout/2,
+                                  debug=True, input_line=custom_inputs)
+        # Wait until the session is dead, then get the output and exit_status
+        # of the session
+        utils_misc.wait_for(lambda: v2v_session.is_alive() is not True, v2v_timeout)
+        cmd_result = process.CmdResult(command=cmd,
+                                       stdout=v2v_session.get_output(),
+                                       exit_status=v2v_session.get_status())
```

Comment:
@chloerh  Please help to update this patch as below, thanks.
        cmd_result = process.CmdResult(command=cmd,
                                       stdout=v2v_session.get_output(),
                                       exit_status=v2v_session.get_status())
        cmd_result.stdout = results_stdout_52lts(cmd_result)
        cmd_result.stderr = results_stderr_52lts(cmd_result)
        return cmd_result